src/style.css

```diff
@@ -0,0 +1,179 @@
+:root {
+  --main-accent: #060b35;
+  --secondary-color: #616070;
+  --white: #fff;
+  --blue-accent: #00acdc;
+  --gray-elements: #e5e5e5;
+  --yellow-stars-color: #ffde6a;
+  --card-padding: 16px;
+}
+
+/* * {
+  box-sizing: border-box;
+} */
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: var(--main-accent);
+}
+
+body {
+  margin: 0;
+}
+
+div {
+  margin: 0;
+}
+
+p {
+  display: flex;
+  margin: 0;
+}
+
+.card {
+  display: flexbox;
+  max-width: calc(198px - (var(--card-padding) * 2));
+  max-height: 400px;
+  padding: var(--card-padding);
+
+  justify-content: center;
+  align-items: center;
+
+  background-color: var(--white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-weight: 500;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__image-iMac {
+  display: flex;
+
+  background-image: url(images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 160px;
+  height: 134px;
+
+  margin: 16px 0 40px;
+  left: calc(50% - (160px / 2));
+}
+
+.card__title {
+  position: relative;
+  display: flex;
+  margin: 0;
+  padding-bottom: 4px;
+
+  align-items: center;
+  line-height: 18px;
+  color: var(--main-accent);
+}
+
+.card__code {
+  display: flex;
+  margin: 0;
+  padding-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--secondary-color);
+}
+
+.card__rate {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+
+  align-items: center;
+  margin: 0;
+  padding-bottom: 24px;
+}
+
+.card__star {
+  display: flex;
+  margin: 0;
+}
+
+.stars-star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.card__stars--4 :nth-child(-n+4) {
```

Comment:
Your stars number should be able to change when you change the modifier in HTML. At the moment your CSS code doesn't support that. It only supports 4 stars.